src/BTVNanoCommissioning/utils/compile_jec.py

```diff
@@ -260,6 +260,36 @@ def jet_factories(campaign):
                 ]
             ),
         },
+        "Summer22EERun3": {
+            "mc": jet_factory_factory(
+                files=[
+                    # "src/BTVNanoCommissioning/data/JME/Winter22Run3/Winter22Run3_V1_MC_PtResolution_AK4PFPuppi.jr.txt",
```

Comment:
Hi @Ming-Yan the JER corrections are also available now, could you please add them too? See [this message](https://cms-talk.web.cern.ch/t/new-jecs-for-run3-prompt-fg-eras/26529/2)